nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -445,6 +445,39 @@ users.users.me =
       </listitem>
     </itemizedlist>
    </listitem>
+   <listitem>
+    <para>
+     The <option>services.xserver.displayManager.auto</option> module has been removed.
+     It was only intended for use in internal NixOS tests, and gave the false impression
+     of it being a special display-manager when it's actually LightDM.
+     Please use the <option>services.xserver.displayManager.lightdm.autoLogin</option> options instead,
+     or any other display-manager in NixOS as they all support auto-login. If you used this module specifically
+     because it permitted root auto-login you can override the lightdm-autologin pam module like:
+<programlisting>
+security.pam.services.lightdm-autologin.text = lib.mkForce ''
```

Comment:
```suggestion
<link xlink:href="opt-security.pam.services._name__.text">security.pam.services.lightdm-autologin.text</link> = lib.mkForce ''
```

Reply:
I actually tried to click on this link on the built note and it didn't work. Any ideas?

Reply:
Sorry, the link is missing `#`.